test/stripe/util_test.rb

```diff
@@ -2,6 +2,22 @@
 
 module Stripe
   class UtilTest < Test::Unit::TestCase
+    context "OPTS_COPYABLE" do
+      should "include :apibase" do
+        assert_include Stripe::Util::OPTS_COPYABLE, :api_base
+      end
+    end
+
+    context "OPTS_PERSISTABLE" do
+      should "include :client" do
+        assert_include Stripe::Util::OPTS_PERSISTABLE, :client
+      end
+
+      should "not include :idempotency_key" do
+        refute_includes Stripe::Util::OPTS_PERSISTABLE, :idempotency_key
+      end
+    end
+
```

Comment:
These tests are probably a little unnecessary, but I added them as a really basic sanity check just because it's so easy to add or remote things from the `OPTS_*` constants.